internal/rbd/rbd_util.go

```diff
@@ -156,6 +156,21 @@ func (rv *rbdVolume) Destroy() {
 	}
 }
 
+// spec returns the image-spec (pool/image) format of the image
+func (rv *rbdVolume) spec() string {
+	return fmt.Sprintf("%s/%s", rv.Pool, rv.RbdImageName)
+}
+
+// spec returns the snap-spec (pool/image@snap) format of the snapshot
+func (rs *rbdSnapshot) spec() string {
+	return fmt.Sprintf("%s/%s@%s", rs.Pool, rs.RbdImageName, rs.RbdSnapName)
+}
+
+// imageSpec returns the image-spec (pool/image) format of the snapshot
+func (rs *rbdSnapshot) imageSpec() string {
+	return fmt.Sprintf("%s/%s@%s", rs.Pool, rs.RbdImageName, rs.RbdSnapName)
```

Comment:
This appears to be the same as rbdSnapshot.spec, unless I'm missing something. I am also unsure that this matches the doc comment.

Reply:
Thanks. I fixed it.

Reply:
fixed.